### ISSUE TITLE
Identification of different (laptop) keyboards with the same keyboard ID

### ISSUE THREAD

**Issue by tkna91** [rvaiya/keyd#526]
My recently acquired new laptop (Fujitsu Lifebook U938/S) had the same keyboard ID `0001:0001` as another laptop I originally owned (Acer C871T-A14P) as shown below.

```
[root@fu1 ~]# hostname
fu1
[root@fu1 ~]# keyd monitor
device added: 0002:0007 SynPS/2 Synaptics TouchPad (/dev/input/event10)
device added: 0001:0001 AT Translated Set 2 keyboard (/dev/input/event3)
[root@fu1 ~]#
```

```
[root@ac1 ~]# hostname
ac1
[root@ac1 ~]# keyd monitor
device added: 06cb:cdb6 Synaptics TM3510-001 (/dev/input/event10)
device added: 0fac:1ade keyd virtual pointer (/dev/input/event9)
device added: 0fac:0ade keyd virtual keyboard (/dev/input/event8)
device added: 0000:0000 Raydium Touchscreen (/dev/input/event7)
device added: 0001:0001 AT Translated Set 2 keyboard (/dev/input/event3)
[root@ac1 ~]#
```
However, I cannot use the same config because of the different key layout. Is there any way to deal with this situation?

**Comment by tkna91** [comment]
I solved the problem by renaming the file in `yadm`, the dot file manager I use, as follows
```
# ls -l /etc/keyd/*hostname*
-rw-r--r-- 1 root root 301 Jul  2 22:51 /etc/keyd/ac1.conf##hostname.ac1
-rw-r--r-- 1 root root 288 Jul  2 22:51 /etc/keyd/fu1.conf##hostname.fu1
#
```

For example, in an environment where the hostname is fu1, a symbolic link file `fu1.conf -> fu1.conf###hostname.fu1` will be created.

https://yadm.io/docs/alternates

Close it as you like.

**Comment by rvaiya** [comment]
I believe this should also be fixed in ca9aa51f1. Device ids now include a unique identifier that is independent of the product/vendor id.
